packages/gcf-utils/src/bin/genkey.ts

```diff
@@ -25,127 +25,133 @@ import { Base64 } from 'js-base64';
 import * as tmp from 'tmp';
 
-let argv = yargs
-    .command('gen', "Create and upload a client key.", (yargs: Argv) => {
-        return yargs.option('keyfile', {
-            describe: "The path to the .pem keyfile",
-            type: 'string',
-            default: "key.pem",
-        }).option('verbose', {
-            alias: 'v',
-            default: false,
-            type: 'boolean'
-        }).option('project', {
-            describe: 'Name of GCP project',
-            alias: 'p',
-            type: 'string',
-            demand: true
-        }).option('location', {
-            alias: 'l',
-            desription: 'Keyring location',
-            type: 'string',
-            default: 'global'
-        }).option('bot', {
-            alias: 'b',
-            type: 'string',
-            description: 'Name of the bot'
-        }).option('bucket', {
-            alias: 'bu',
-            type: 'string',
-            description: 'Name of the Bucket'
-        }).option('id', {
-            alias: 'i',
-            type: 'string',
-            description: 'ID of the GitHub Application'
-        });
-    }).argv;
-
-
-const keyfile: string = argv.file as string || 'key.pem';
+const argv = yargs.command(
+  'gen',
+  'Create and upload a client key.',
+  (yargs: Argv) => {
+    return yargs
+      .option('keyfile', {
+        describe: 'The path to the .pem keyfile',
+        type: 'string',
+        default: 'key.pem',
+      })
+      .option('verbose', {
+        alias: 'v',
+        default: false,
+        type: 'boolean',
+      })
+      .option('project', {
+        describe: 'Name of GCP project',
+        alias: 'p',
+        type: 'string',
+        demand: true,
+      })
+      .option('location', {
+        alias: 'l',
+        desription: 'Keyring location',
+        type: 'string',
+        default: 'global',
+      })
+      .option('bot', {
+        alias: 'b',
+        type: 'string',
+        description: 'Name of the bot',
+      })
+      .option('bucket', {
+        alias: 'bu',
+        type: 'string',
+        description: 'Name of the Bucket',
+      })
+      .option('id', {
+        alias: 'i',
+        type: 'string',
+        description: 'ID of the GitHub Application',
+      });
+  }
+).argv;
+
+const keyfile: string = (argv.file as string) || 'key.pem';
 const project: string = argv.project as string;
-const location: string = argv.location as string || 'global';
-const keyring: string = argv.keyring as string || 'probot-keys';
+const location: string = (argv.location as string) || 'global';
+const keyring: string = (argv.keyring as string) || 'probot-keys';
 const bucketName: string = argv.bucket as string;
 const botname: string = argv.bot as string;
 const webhookSecret: string = argv.secret as string;
 const id: number = argv.id as number;
 
-
 if (!project) {
-    console.error('Project name is required');
-    yargs.showHelp();
-    process.exit(1);
+  console.error('Project name is required');
```

Comment:
Pretty sure there's a way to make `yargs` do this kind of validation for us, right @bcoe ?